rules/repositories.bzl

```diff
@@ -53,13 +53,15 @@ def rules_ios_dependencies():
         sha256 = "1fff3fa1e565111a8f678b4698792101844f57b2e78c5e374431d0ebe97f6b6c",
     )
 
+    # Note: this is a branch of the rules_swift PR
+    # https://github.com/bazelbuild/rules_swift/pull/567
```

Comment:
Do you plan to merge this PR after merging the rules_swift PR?

Reply:
@congt we ended up deciding to to use a short lived fork of these dependencies to let this bake and reap the benefits while upstream refactoring is worked out.

Reply:
I'm so sorry, I think there was some miscommunication in that meeting on Wednesday. I talked with @segiddins and @amberdixon and we'd heard different things, so my apologies for not ensuring we were all on the same page about next steps.

I'm *super* excited to get this landed and the immediate impact it will have. Is there an impediment to upstreaming the changes, or is it the work to address PR feedback and gathering consensus on the approach? Are we concerned there will be pushback overall?

In general, we've taken the approach of attempting to upstream changes first, and only resorting to forking if we can't get the changes landed. If we hit a wall upstreaming the changes, then we should reevaluate. My perspective is that it is harder to unfork things as time goes on, and there isn't as much incentive to unfork, which is more likely to lead to a fork becoming long lived (which has associated maintenance cost). I'm aware that this means it takes longer to realize the impact of the work. I'm definitely open to other perspectives here, both from you and the rest of the team.

Reply:
I agree to getting this upstreamed and want to get it working for all Bazel users. We should try to work with the maintainers of rules_swift to get it landed. That being said, the PR has been open at rules_swift since last Friday which should be upstream able. I also mentioned the short lived fork in our plan to move this forward https://github.com/bazel-ios/rules_ios/pull/212/files#diff-26cd539e3b611f02c1703c8fce4656a533b33dc08a4479ab5bfa7523260fa292R123 

@segiddins can you list the required changes to land this? There's been a lot of ideas shared and back and forth on the PR

Reply:
@justinseanmartin Here is what I propose for the next steps: 
1. keep the temp fork of rules_swift/index-import until everyone feels good about landing the PR ( and weeks of testing internally )
2. land the PR in rules_ios that points at the forks of index-import and rules_swift
3. point repos at HEAD of rules_ios after the PR lands

Overall we might want to have a larger discussion with the community around landing PRs and features into rules_swift behind feature flags. It's a great way to share ideas. Not everyone has access to work that way ATM.

Reply:
Overall, we're in agreement with the commitment to actively work to get  unforked in the near term (weeks not months), taking an active role in reaching out and collaborating with `rules_swift` and `index-import` maintainers.

Reply:
@justinseanmartin SGTM -  I'm fully in support to getting this un-forked in some form 👍 

Longer term I hope we can get a flag to handle this in swift / clang: that'd be the ideal state for this specific issue. The main reason we need to the helper binary ( e.g. my PR to `index-import` ) in `rules_swift` is that we can't tell the compiler to index only impl files. We'll also need to wait for that release cycle if we go with the swift/clang approach, that is if we could get it landed in there.